controller/model/edge_router_handlers_test.go

```diff
@@ -25,18 +25,17 @@ func (ctx *TestContext) testGetEdgeRoutersForServiceAndIdentity(*testing.T) {
 	ctx.requireNewEdgeRouterPolicy(ss("#all"), ss("#all"))
 
 	// test default case, with no limits on service
-	ctx.True(ctx.isEdgeRouterAccessible(edgeRouter.Id, identity.Id, service.Id))
-	ctx.True(ctx.isEdgeRouterAccessible(edgeRouter2.Id, identity.Id, service.Id))
+	ctx.False(ctx.isEdgeRouterAccessible(edgeRouter.Id, identity.Id, service.Id))
```

Comment:
Why would these switch to false?

Reply:
In the old model if you didn't specify edgeRouterRoles on the service, the service had access to all edge routers. 
In the new model a service needs to be added to at least one service edge router policy in order to have access to any edge routers